modules/apps/jaggery/mdm/src/mdm/modules/backendServiceInvoker.js

```diff
@@ -135,14 +135,19 @@ var backendServiceInvoker = function () {
     function initiateWSRequest(action, endpoint, payload, successCallback, errorCallback, soapVersion) {
         var ws = require('ws');
         var wsRequest = new ws.WSRequest();
-        var options = [];
+        var options = new Array();
         if (IS_OAUTH_ENABLED) {
             var accessToken = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER).accessToken;
-            if (!(!accessToken.trim())) {
-                var headers = [{name: constants.AUTHORIZATION_HEADER, value: constants.BEARER_PREFIX + accessToken}];
+            if (!(!accessToken)) {
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158